### DISCUSSION TITLE
Cloud Providers

### DISCUSSION THREAD

**Discussion by nferrari** [spiral-project/ihatemoney#997] [Show and tell]
Hi guys, 

Here is my recent question (in French) to Alexis, to which he suggested me to open a discussion:

    Salut Alexis,

    Je reprend sur ce ticket qui concerne Ihatemoney.

    Je ne sais pas si tu as vu, on a créé une Marketplace sur notre site, pour faciliter l'install d'application avant même d'avoir un profil sur alwaysdata. Bref, il y a ihatemoney dans la liste, puisque que c'était déjà dispo en installation 1click.

    Je suis en train de regarder pour chaque app s'il y a une page qui référence les providers, c'est souvent le cas. Par exemple :

        Django-CMS, une page sur leur site : https://www.django-cms.org/en/hosting-services/
        Gitea, une page de la doc : https://docs.gitea.io/en-us/install-on-cloud-provider/

    D'après ce que je vois, il pourrait y avoir une page sur votre doc. Qu'en penses-tu ? Je peux faire une PR évidemment.

    Merci pour ton retour ;)

Here is our Marketplace I'm talking about: https://www.alwaysdata.com/en/marketplace/
Obviously, if you want to stay neutral, or add other providers to the list, no problem, this is your call ;)
And FYI, IHM can be installed on our free plan.

Cheers!

**Comment by almet** [comment]
I think it's okay to point to alwaysdata services, it's a simple way to deploy, si it could be listed as an alternative to docker for instance. 

What do you think @Glandos @Natim @zorun ?

**Reply by Natim** [reply]
I'm ok with this yes !

**Reply by Glandos** [reply]
No issue for me on this side.
If you submit a PR, I think it could be both mentioned in the docs available on https://ihatemoney.readthedocs.io/en/latest/ and in the README.md since it also list the hosted version.

**Comment by nferrari** [comment]
Thank you guys, will do!
